### PR TITLE
Ellipsis in smart contracts

### PR DESCRIPTION
**Pull request by netoneko**
## Description

Prerequisite for #1225 

- [x] Adds ellipsis to smart contracts

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I have read the [CONTRIBUTING](https://github.com/orbs-network/orbs-contributor-guide/blob/master/CONTRIBUTING.md) doc
- [ ] I have opened an issue and discussed the solution suggested in this PR
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: services/processor/native/test/call_argument_test.go [hunk 1]
```diff
@@ -75,7 +75,6 @@ func TestProcessCall_Arguments(t *testing.T) {
 					require.NoError(t, err, "call should succeed")
 					require.Equal(t, tt.expectedOutput, output.OutputArgumentArray, "call return args should be equal")
 				}
-				require.Equal(t, tt.expectedResult, output.CallResult, "call result should be equal")
```
Comment (electricmonk) [inline services/processor/native/test/call_argument_test.go]: why did you remove this line?

File: services/processor/native/call_test.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	args := argsToArgumentArray([]byte("one"), []byte("two"))
+
+	inValues, err := s.prepareMethodInputArgsForCall(methodInstance, args, "funcName")
+	require.NoError(t, err)
+
+	outValues := reflect.ValueOf(methodInstance).Call(inValues)
+	require.EqualValues(t, [][]byte{[]byte("one"), []byte("two")}, outValues[0].Interface())
+}
+
+func TestPrepareMethodArgumentsForCallWithIncorrectNumberOfArgs(t *testing.T) {
```
Comment (electricmonk) [inline services/processor/native/call_test.go]: this one is "too many arguments". Do we need one for "too few"?

File: services/processor/native/test/call_argument_test.go (hunk 1 above)
Comment (netoneko) [reply services/processor/native/test/call_argument_test.go]: Because it's not right to assume that if we pass X arguments to the transformer method, it should also return X arguments.

File: services/processor/native/call_test.go (hunk 2 above)
Comment (netoneko) [reply services/processor/native/call_test.go]: It is covered by other, more high level tests here: https://github.com/orbs-network/orbs-network-go/blob/656c5b03ada55c68ab51d5f0b34d737184f7448f/services/processor/native/test/call_argument_test.go#L41

**Comment by netoneko** [comment]
@electricmonk there is an e2e in the next pr: https://github.com/orbs-network/orbs-network-go/pull/1225/files#diff-f5e03d1a398e9820ddebcd51c578af99
